### PR TITLE
[test] Update compiler flag for Gperftools check

### PR DESCRIPTION
**Pull request by jgphpc**
Will fix UES-606

### CODE REVIEW

**Comment by vkarak** [comment]
@jgphpc Test is failing on Dom. Is this expected?

**Comment by jgphpc** [comment]
@jenkins-cscs retry dom

File: cscs-checks/tools/profiling_and_debugging/gperftools_mpi_omp.py [hunk 1]
```diff
@@ -72,14 +76,21 @@ def __init__(self, lang):
...
             sn.assert_found('PDF document', self.rpt_file_doc),
         ])
+        self.perf_patterns = {
+            'jacobi_elapsed': self.report_flat_pctg,
+        }
+        self.reference = {
+            '*': {
+                'jacobi_elapsed': (0, None, None, '%'),
+            }
+        }
```
Comment (vkarak) [inline cscs-checks/tools/profiling_and_debugging/gperftools_mpi_omp.py]: You don't need the star anymore. ReFrame will automatically generate it, if it doesn't find it.

File: cscs-checks/tools/profiling_and_debugging/gperftools_mpi_omp.py [hunk 2]
```diff
@@ -88,3 +99,12 @@ def setup(self, environ, partition, **job_opts):
...
         self.build_system.ldflags = flags + ['`pkg-config --libs libprofiler`']
+
+# {{{
+    @property
+    @sn.sanity_function
+    def report_flat_pctg(self):
+        regex = r'^\s+\d+ms\s+(?P<pctg>\d+.\d+)%.*_jacobi.\w+:\d+'
+        result = sn.extractsingle(regex, self.rpt_file_txt, 'pctg', float)
+        return result
+# }}}
```
Comment (vkarak) [inline cscs-checks/tools/profiling_and_debugging/gperftools_mpi_omp.py]: What is this?

File: cscs-checks/tools/profiling_and_debugging/gperftools_mpi_omp.py [hunk 3]
```diff
@@ -72,14 +76,21 @@ def __init__(self, lang):
...
+            'jacobi_elapsed': self.report_flat_pctg,
+        }
+        self.reference = {
+            '*': {
+                'jacobi_elapsed': (0, None, None, '%'),
+            }
+        }
         self.maintainers = ['JG']
-        self.tags = {'production'}
+        self.tags = {'performance'}
```
Comment (vkarak) [inline cscs-checks/tools/profiling_and_debugging/gperftools_mpi_omp.py]: No more production?

File: cscs-checks/tools/profiling_and_debugging/gperftools_mpi_omp.py (hunk 2 above)
Comment (jgphpc) [reply cscs-checks/tools/profiling_and_debugging/gperftools_mpi_omp.py]: https://www.linux.com/tutorials/vim-tips-folding-fun/

File: cscs-checks/tools/profiling_and_debugging/gperftools_mpi_omp.py (hunk 1 above)
Comment (jgphpc) [reply cscs-checks/tools/profiling_and_debugging/gperftools_mpi_omp.py]: can you add a comment in https://github.com/eth-cscs/reframe/blob/master/docs/running.rst#generating-a-performance-report ?

Comment (vkarak) [reply cscs-checks/tools/profiling_and_debugging/gperftools_mpi_omp.py]: It is mentioned in the tutorial: https://github.com/eth-cscs/reframe/pull/866/files

**Pull request merged**